release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -298,6 +298,89 @@ With this release, the `MaxUnavailableStatefulSet` featureset configuration para
+
+[id="ocp-4-14-monitoring-new-option-to-create-alerts-based-on-core-platform-metrics"]
+==== New option to create alerts based on core platform metrics
+
+With this release, administrators can create new alerting rules based on core platform metrics.
+You can now modify settings for existing platform alerting rules by adjusting thresholds and by changing labels.
+You can also define and add new custom alerting rules by constructing a query expression based on core platform metrics in the `openshift-monitoring` namespace.
+This feature was included as a technology preview in a previous {product-title} release and is now generally available.
+For more information, see xref:../monitoring/managing-alerts.adoc#managing-core-platform-alerting-rules_managing-alerts[Managing alerting rules for core platform monitoring].
+
+[id="ocp-4-14-monitoring-new-option-to-specify-resource-limits-for-all-monitoring-components"]
+==== New option to specify resource limits for all monitoring components
+
+With this release, you can now specify resource requests and limits for all monitoring components, including the following:
+
+* Alertmanager
+* kube-state-metrics
+* node-exporter
+* openshift-state-metrics
+* Prometheus
+* Prometheus Operator and its Admission webhook
+* Telemeter Client
+* Thanos Querier
+* Thanos Ruler
+
+In previous versions of {product-title}, you could only set options for Prometheus, Alertmanager, Thanos Querier, and Thanos Ruler.
+
+[id="ocp-4-14-monitoring-new-options-to-configure-node-exporter-collectors"]
+==== New options to configure node-exporter collectors
+
+With this release, you can customize Cluster Monitoring Operator (CMO) config map settings for additional node-exporter collectors.
+The following node-exporter collectors are now optional, and you can enable or disable each one individually by using config map settings:
+
+* `gomaxprocs` collector
+* `hwmon` collector
+* `ksmd` collector
+* `mountstats` collector
+* `ntp` collector
+
+In addition, you can now exclude network devices from the relevant collector configuration for the `netdev` and `netclass` collectors.
+
+[id="ocp-4-14-monitoring-new-options-to-deploy-monitoring-web-console-plugin-resources"]
+==== New option to deploy monitoring web console plugin resources
+
+With this release, the monitoring pages in the *Observe* section of the {product-title} web console are deployed as a xref:../web_console/dynamic-plugin/overview-dynamic-plugin.adoc[dynamic plugin].
+With this change, you can now use Cluster Monitoring Operator (CMO) configurations to deploy certain {product-title} web console monitoring plugin resources.
+Specifically, you can use CMO settings to configure the following monitoring stack features:
+
+* Node tolerations
+* Topology spread constraints
+* Resource requests
+* Resource limits
```

Comment:
based on https://github.com/openshift/cluster-monitoring-operator/blob/release-4.14/pkg/manifests/types.go#L688-L697, it is better to update to
```suggestion
* Node selectors
* Node tolerations
* Topology spread constraints
* Resource requests
* Resource limits
```